ui/docs/tutorial/apis.md

```diff
@@ -9,11 +9,11 @@ followup:
 
 This section shows you how to access Taskcluster APIs programmatically.  The
 examples use JavaScript (ES6) using the JavaScript [taskcluster
-client](https://github.com/taskcluster/taskcluster-client) but apply to [other
+client](https://github.com/taskcluster/taskcluster/tree/master/clients/client) but apply to [other
 supported
 languages](/docs/manual/using/integration/libraries).
 
-Note: the graphical Taskcluster tools on `tools.taskcluster.net` are all
+Note: the graphical Taskcluster tools on `https://taskcluster-ui.herokuapp.com` are all
```

Comment:
@djmitche Is the reference to https://taskcluster-ui.herokuapp.com fine? We can remove this section if it is no big deal.

Reply:
I think we can just remove this paragraph -- the frontend doesn't use tc-client-web, anyway!

Reply:
@djmitche Done!